src/atframe.jl

```diff
@@ -30,9 +30,9 @@ function getclip!(animation::Animation, path::Path)
 end
 
 js_quaternion(m::AbstractMatrix) = js_quaternion(RotMatrix(SMatrix{3, 3, eltype(m)}(m)))
-js_quaternion(q::UnitQuaternion) = [q.x, q.y, q.z, q.w]
-js_quaternion(::UniformScaling) = js_quaternion(UnitQuaternion(1., 0., 0., 0.))
-js_quaternion(r::Rotation) = js_quaternion(UnitQuaternion(r))
+js_quaternion(q::QuatRotation) = [q.q.v1, q.q.v2, q.q.v3, q.q.s]
```

Comment:
I think we should make functions for this--if we have to do some ugly field access, let's at least wrap it up in a function so we only have to change one thing when Rotations.jl changes again. Normally I don't like the extra `get_` prefix, but having a function just called `w()` seems a bit too terse. So how about we define 

```julia
get_w(q::QuatRotation) = q.q.v1
```

etc? We could do the same thing over in RigidBodyDynamics.jl (we'll have to copy those functions, but I think that's still better than putting even more field access in).

Reply:
We have a couple of options to choose from:

1. We can write our own wrappers (as you suggested):
    ```julia
    get_w(q::QuatRotation) = q.q.s
    get_x(q::QuatRotation) = q.q.v1
    get_y(q::QuatRotation) = q.q.v2
    get_z(q::QuatRotation) = q.q.v3
    ```
    and then redefine `js_quaternion(q::QuatRotation)` as
    ```julia
    js_quaternion(q::QuatRotation) = [get_w(q), get_x(q), get_y(q), get_z(q)]
    ```

2. Convert to a `Quaternions.jl` quaternion:
    ```julia
    function js_quaternion(q::QuatRotation)
        q_ = Quaternions.Quaternion(q)
        return [q_.v1, q_.v2, q_.v3, q_.s]
    end
    ```
    (Requires adding and importing `Quaternions.jl` at its top level.)

3. Get the underlying parameters via `Rotations.params`:
    ```julia
    function js_quaternion(q::QuatRotation)
        w, x, y, z = Rotations.params(q)
        return [x, y, z, w]
    end
    ```
    (Requires taking care about the return order of `Rotations.params(q)`.)

Reply:
I agree we should try to keep [RigidBodyDynamics.jl](https://github.com/JuliaRobotics/RigidBodyDynamics.jl) and [MeshCat.jl](https://github.com/rdeits/MeshCat.jl) consistent. I'll update https://github.com/JuliaRobotics/RigidBodyDynamics.jl/pull/629 with whichever option we end up choosing.

Reply:
Oh, I didn't know about `Rotations.params`. I definitely like that option the best, since it doesn't require any field access at all. Good idea!

Reply:
I too like option 3 the most. I have now updated this PR as well as https://github.com/JuliaRobotics/RigidBodyDynamics.jl/pull/629.